### PR TITLE
US2.1 / 2.2 : Add / edit financial information

### PR DESCRIPTION
**Pull request by iStyyx**

### CODE REVIEW

**Review by mcbeloy — APPROVED** [review]

File: templates/financial_info/edit.html.twig [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+{% endblock header %}
+
+{% block main %}
+<div class="generalInfo">
+
+  {% block formTitle %}Edit booking # {{ booking.reference }} {% endblock formTitle %}
+
+  {% block generalInfo %}
+  {{ include('financial_info/_form.html.twig', {'button_label': 'Update'}) }}
+  {% endblock generalInfo %}
```
Comment (RomainClair) [inline templates/financial_info/edit.html.twig:14]: The edit.html.twig is a final template.
It should not create new block as no template will fill them

File: templates/financial_info/edit.html.twig [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}New Booking{% endblock %}
+{% block header %}
+{% endblock header %}
```
Comment (RomainClair) [inline templates/financial_info/edit.html.twig:5]: Why completing a block only with spaces and new lines ?

**Review by RomainClair — CHANGES_REQUESTED** [review]

File: templates/financial_info/edit.html.twig (hunk 2 above)
Comment (iStyyx) [reply templates/financial_info/edit.html.twig]: It was for the future navbar, but it's removed now.

**Review by soufcod3 — APPROVED** [review]

**Review by soufcod3 — APPROVED** [review]
Looking good

**Review by RomainClair — CHANGES_REQUESTED** [review]
You should not commit files you don't know anything about
You should validate user input using assertions in the Booking entity

**Review by mcbeloy — APPROVED** [review]

**Comment by mcbeloy** [comment]
Good

**Review by soufcod3 — APPROVED** [review]
The validation constraints in the Booking entity seem good to me

**Review by RomainClair — DISMISSED** [review]

**Review by alexandregrue — APPROVED** [review]

**Review by RomainClair — APPROVED** [review]

**Pull request merged**